scripts/lighthouseBudget.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+
+const getCategoryScores = data => {
+  const { categories } = data;
+
+  return {
+    ally: categories.accessibility.score * 100,
+    bestPractises: categories['best-practices'].score * 100,
+    seo: categories.seo.score * 100,
+  };
+};
+
+const isAboveThreshold = (scoreValue, budgetValue) => {
+  if (scoreValue >= budgetValue) {
+    return true;
+  }
+  return false;
+};
+
+const logRow = (category, scoreValue, budgetValue, result) => ({
+  category,
+  scoreValue,
+  budgetValue,
+  result,
+});
+
+const compareToBudget = (categories, scoreResult, scoreBudget) => {
+  let result = true;
+  const logArray = [];
+  // eslint-disable-next-line consistent-return
+  categories.forEach(prop => {
+    const test = isAboveThreshold(scoreResult[prop], scoreBudget[prop]);
+    const b = logRow(prop, scoreResult[prop], scoreBudget[prop], test);
```

Comment:
Please consider using a more descriptive variable name here.

Reply:
Yea good point fixing in now